.github/01_bugreport.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Bug Report
+about: Report a bug to help us improve MSBuild.
+title: ''
+labels:
+---
+
+<!-- This is a template that helps us provide quicker feedback. Please use any relevant sections and delete anything you don't need. -->
+
+### Issue Description
+<!--
+Please include a clear and concise description of the problem.
+-->
+
+### Expected Behavior
+<!--
+* Include the expected output or behavior.
+-->
+
+### Actual Behavior
+<!--
+* Include the actual output or behavior.
+-->
+
+### Analysis
+<!--
+* If you have an idea where the problem might lie, let us know that here.
+* Please include any pointers to code, relevant changes, or related issues you know of.
+* If you don't know, you can delete this section.
+-->
+
+### Versions & Configurations
+<!--
+* In a visual studio developer command prompt, run `msbuild -version` and paste the output here.
+* If applicable, version of the tool that invokes MSBuild (Visual Studio, dotnet CLI, etc):
+
+Post any other relevant configuration settings here.
+* OS, architecture, etc.
+-->
+
+### Steps to Reproduce
+<!--
+Include as much of the following if possible:
+
+* A minimal sample project that reproduces the issue.
+* Your zipped project.
+* IDE / CLI steps to create the project and reproduce the behaviour.
+* Your command line invocation
+-->
+
+### Attach a binlog
+<!--
+To help us figure out and fix the issue sooner, please include a binlog.
```

Comment:
Binlogs can contain sensitive information (eg., environment variables, usernames, paths, conceiveably secrets if they're passing them insecurely.., project names, file names.. )

Should you warn clearly here that they should not attach anything they don't want to be public? 
And also remind them that they can open in the bin log viewer to see what info is in it?
Is there a secure way to send logs? Might be worth describing it here.

Reply:
Ah, of course this will be in the links you've shared, but perhaps worth a reminder here about privacy.

Reply:
I'll add a notice for the risk of posting a binlog.